e2e/mock-api/jest.config.ts

```diff
@@ -12,5 +13,5 @@ module.exports = {
   },
   moduleFileExtensions: ['ts', 'js', 'html'],
   coverageDirectory: '../../coverage/samples/mock-api',
-  preset: '../../jest.preset.ts',
+  preset: '../../jest.preset.js',
```

Comment:
I'm seeing some Jest files as `.ts` and some as `.js`. Is there something preventing us from going full one or the other?

Reply:
yeah, jest doesnt like the preset as typescript. idk why.

Reply:
I feel like these files should be one or the other and not a mixed bag of file types. Could we just make all Jest files `.js`?